components/mate/caja/Makefile

```diff
@@ -43,6 +43,7 @@ CONFIGURE_OPTIONS+=	--disable-static
 CONFIGURE_OPTIONS+=	--disable-scrollkeeper
 CONFIGURE_OPTIONS+=	--localstatedir=/var/lib
 CONFIGURE_OPTIONS+=	--with-gnome-distributor="The OpenIndiana Project"
+CONFIGURE_OPTIONS+=     --libexecdir=/usr/lib/$(COMPONENT_NAME)
```

Comment:
I dont know. But $(CONFIGURE_LIBDIR.$(BITS))/$(COMPONENT_NAME) will definitely not make conflicts, thus i thought it the better option.